insonmnia/node/tokens.go

```diff
@@ -38,6 +39,30 @@ func (t *tokenAPI) Balance(ctx context.Context, _ *sonm.Empty) (*sonm.BalanceRep
 	}, nil
 }
 
+func (t *tokenAPI) Deposit(ctx context.Context, amount *sonm.BigInt) (*sonm.Empty, error) {
+	if err := t.remotes.eth.MasterchainToken().IncreaseApproval(ctx, t.remotes.key, blockchain.GatekeeperMasterchainAddr(), amount.Unwrap()); err != nil {
+		return nil, errors.WithMessage(err, "cannot change allowance")
```

Comment:
AFAIR errors.WithMessage also have quite strange formatting in attributes.

Reply:
No, it [doesn't](https://github.com/pkg/errors/blob/master/errors.go#L228), just a colon-separated string.